src/Mod/Fem/App/FemPostPipeline.cpp

```diff
@@ -256,7 +258,7 @@ void FemPostPipeline::load(FemResultObject* res) {
 
     //Now copy the point data over
     //############################
-    if(res->getPropertyByName("Velocity")){
+    if(res->getPropertyByName("Velocity")){  // consider better way to detect result type, res->Type == "CfdResult"

```

Comment:
Maybe a **TODO:** at the beginning of the comment would be useful for future friendly searches?